content/docs/user-guide/how-to/undo-dvc-add.md

```diff
@@ -0,0 +1,47 @@
+# Undo dvc add
+
+There are situations where you `dvc add` a data file by mistake and want DVC to
+stop tracking that file. Follow the steps listed here to undo `dvc add`.
```

Comment:
Mistakes are not the main case where this is needed. Probably no need to mention "by mistake".